.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/realm/SwiftLint
+    rev: 0.50.3
+    hooks:
+      - id: swiftlint
+      - entry: swiftlint --fix
```

Comment:
How does this work?

Reply:
After I finish with the changes, I will create a document explaining the steps so everyone can be aware of how it works and what needs to be installed.

Reply:
But can you explain it in the pull request too? Pull requests are also a form of documentation and it's helpful to be able to review it.

Reply:
Just updated the PR with more information

Reply:
So you have to install an additional tool on the local machine for this to work?

Reply:
Yes I updated the PR description with the steps!